R/recipe_read.R

```diff
@@ -210,6 +210,21 @@ recipe_read_check_resources <- function(x, filename, path) {
   if (is.null(x)) {
     return(NULL)
   }
+
+  # make sure that a directory resource does not have a trailing /
+  # There is much more sanitation that could be done here...
+  is_dir <- is_directory(file.path(path, x))
```

Comment:
I think either this should be ok (no warning, strip the slash and continue) or not ok (throw an error) but not a warning - either we want the user to do something about it or not

Reply:
Warning removed - We might want to add more file name sanitation; no one is going to read dozens of warnings